liquibase/changes/feat_741.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE OfficialStatusHistory(
+    is_official BOOLEAN NOT NULL,
+    feed_id VARCHAR(255) NOT NULL,
+    contact_email VARCHAR(255) NOT NULL,
```

Comment:
For clarification, this is the email of the person stamping the official property, not the official contact email.

Reply:
should i rename it? any suggestion?

Reply:
done ✅